spec/controllers/tasks_controller_spec.rb

```diff
@@ -28,5 +28,28 @@
             get :index
             expect(JSON.parse(response.body).size).to eq(2)
```

Comment:
Adding a JSON presenter helper class could help in the test duplication as well. 😇

Reply:
Even if I have a JsonPresenterHelper I wouldn't use it in the tests as well. For me personally it will be easier to read JSON.parse() because I already know what it will do then to find a custom method  instead and need to check another folder to see what  that method will do. But this is just my preference and if my point of view doesn't make sense I will definitely be open for the best practice 😊

Reply:
I thought about this when I gave the feedback. Its not really shortening up the code at all, youre absolutely right. I think I mentioned it before, that sometimes these suggestions in small apps like this dont make sense. But think about your todo app becoming a very large app, say 8 million lines of code... Getting used to abstracting logic into reusable components or classes is a good habit. I am comfortable with you leaving it how you have it though. Thanks for the discussion.

Reply:
Thank you for your feedback .